libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -93,6 +102,10 @@ class AP_BattMonitor
     float voltage(uint8_t instance) const;
     float voltage() const { return voltage(AP_BATT_PRIMARY_INSTANCE); }
 
+    /// get voltage with sag removed (based on battery current draw and resistance)
+    float voltage_resting_estimate(uint8_t instance) const;
```

Comment:
Nitpicking: to be more correct and match other methods make _instance_  method parameter _const_